instruments/abstract_instruments/instrument.py

```diff
@@ -525,6 +526,26 @@ def open_serial(cls, port=None, baud=9600, vid=None, pid=None,
         return cls(ser)
 
     @classmethod
+    def open_serial_remote(cls, url=None, baud=115200, dsrdtr=False, rtscts=False,
```

Comment:
I somewhat agree, but I'm also of mixed minds. The API that PySerial offers for local and remote serial ports is subtly different enough that it might make ``open_serial`` confusing to call if it does both. Would something like ``remote=True`` be enough, do you think? If so, we could do both by having the old ``open_serial`` and new ``open_serial_remote`` both basically be private methods called by a new combined ``open_serial``.